cai_causal_graph/causal_graph.py

```diff
@@ -1947,7 +1973,7 @@ def copy(self, include_meta: bool = True) -> CausalGraph:
         :return: A copy of the `cai_causal_graph.causal_graph.CausalGraph` instance.
         """
         graph_dict = self.to_dict(include_meta=include_meta)
-        new_graph = self.__class__.from_dict(graph_dict)
+        new_graph = self.__class__.from_dict(graph_dict, validate=False)
```

Comment:
Why `False`? One could have a graph with cycles but I guess copy is fine with that.